testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/HttpServerTestOptions.java

```diff
@@ -117,6 +118,13 @@ public HttpServerTestOptions setHasExceptionOnServerSpan(
     return this;
   }
 
+  @CanIgnoreReturnValue
+  public HttpServerTestOptions setHasResponseCustomizer(
+      Predicate<ServerEndpoint> hasResponseCustomizer) {
+    this.hasResponseCustomizer = hasResponseCustomizer;
```

Comment:
Is there really a need for a `Predicate`? Is there any scenario where a `boolean` wouldn't work?

Reply:
This can be useful in cases where there is no specific support for an appserver directly, so some built-in error pages might not be handled, but we still expect the servlet fallback to handle it for most endpoints.